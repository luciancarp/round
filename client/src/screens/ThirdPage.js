import React, { Component } from 'react'
import StyledPage from '../components/layout/StyledPage'
import StyledTitle from '../components/layout/StyledTitle'

class ThirdPage extends Component {
  constructor (props) {
    super(props)
    this.state = {
      title: 'Third Page'
    }
  }

  render () {
    return (
      <StyledPage>
        <StyledTitle>{this.state.title}</StyledTitle>
        <p>Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.
        </p>
      </StyledPage>
    )
  }
}

export default ThirdPage
