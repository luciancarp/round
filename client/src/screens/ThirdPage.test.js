import React from 'react'
import { shallow } from 'enzyme'
import ThirdPage from './ThirdPage'

it('should render correctly with no props', () => {
  const component = shallow(<ThirdPage />)
  expect(component).toMatchSnapshot()
})
