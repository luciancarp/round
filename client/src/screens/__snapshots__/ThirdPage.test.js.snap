// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with no props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ThirdPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <ForwardRef>
          Third Page
        </ForwardRef>,
        <p>
          Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Third Page",
        },
        "ref": null,
        "rendered": "Third Page",
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bwzfXH",
            "isStatic": true,
            "rules": Array [
              "
  color: ",
              "#333333",
              ";
  font-size: ",
              "32",
              "px;
",
            ],
          },
          "displayName": "styled.h1",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bwzfXH",
          "target": "h1",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.",
        },
        "ref": null,
        "rendered": "Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.",
        "type": "p",
      },
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  /* margin: auto; */
  margin-left: auto;
  margin-right: auto;
  margin-bottom: 0px;
  padding: ",
          "14",
          "px;
  min-height: calc(100vh - 225px);
  /* text-align: center; */

  @media ",
          "(min-width: 1024px)",
          " {  
    max-width: 1024px;
  }

  @media ",
          "(min-width: 2560px)",
          " {
    max-width: 1400px;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <ForwardRef>
            Third Page
          </ForwardRef>,
          <p>
            Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Third Page",
          },
          "ref": null,
          "rendered": "Third Page",
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bwzfXH",
              "isStatic": true,
              "rules": Array [
                "
  color: ",
                "#333333",
                ";
  font-size: ",
                "32",
                "px;
",
              ],
            },
            "displayName": "styled.h1",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bwzfXH",
            "target": "h1",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.",
          },
          "ref": null,
          "rendered": "Proin accumsan venenatis lectus, nec sollicitudin magna laoreet non. Cras porttitor, risus efficitur tincidunt vehicula, sapien lacus scelerisque diam, quis pretium lacus justo sit amet ante. Vivamus laoreet fringilla libero quis congue. Nunc nisl nunc, feugiat ut lectus ac, porta vehicula sem. Donec quis sollicitudin purus. Curabitur porttitor neque sed ipsum viverra, non congue odio venenatis. Nunc aliquet sem vitae justo tincidunt, ac rutrum purus bibendum. Aenean dictum pretium magna, et viverra arcu blandit id. Phasellus sit amet metus arcu. Cras vehicula leo vitae arcu mattis posuere. Duis viverra quis purus at pharetra. Proin ut lectus nec ipsum viverra maximus vitae eget orci. Sed vehicula mi neque, ut fringilla dolor congue vitae.",
          "type": "p",
        },
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  /* margin: auto; */
  margin-left: auto;
  margin-right: auto;
  margin-bottom: 0px;
  padding: ",
            "14",
            "px;
  min-height: calc(100vh - 225px);
  /* text-align: center; */

  @media ",
            "(min-width: 1024px)",
            " {  
    max-width: 1024px;
  }

  @media ",
            "(min-width: 2560px)",
            " {
    max-width: 1400px;
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
